rules/windows/process_creation/win_netsh_fw_add_susp_image.yml

```diff
@@ -28,27 +28,27 @@ detection:
             - program=
     susp_image:
         CommandLine|contains:
-            - '*%TEMP%*'
-            - '*:\RECYCLER\\*'
-            - '*C:\$Recycle.bin\\*'
-            - '*:\SystemVolumeInformation\\*'
-            - 'C:\\Windows\\Tasks\\*'
-            - 'C:\\Windows\\debug\\*'
-            - 'C:\\Windows\\fonts\\*'
-            - 'C:\\Windows\\help\\*'
-            - 'C:\\Windows\\drivers\\*'
-            - 'C:\\Windows\\addins\\*'
-            - 'C:\\Windows\\cursors\\*'
-            - 'C:\\Windows\\system32\tasks\\*'
-            - '*C:\Windows\Temp\\*'
-            - '*C:\Temp\\*'
-            - '*C:\Users\Public\\*'
-            - '%Public%\\*'
-            - '*C:\Users\Default\\*'
-            - '*C:\Users\Desktop\\*'
-            - '*\Downloads\\*'
-            - '*\Temporary Internet Files\Content.Outlook\\*'
-            - '*\Local Settings\Temporary Internet Files\\*'
+            - '%TEMP%'
+            - ':\RECYCLER\\'
+            - 'C:\$Recycle.bin\\'
+            - ':\SystemVolumeInformation\\'
+            - 'C:\\Windows\\Tasks\\'
+            - 'C:\\Windows\\debug\\'
+            - 'C:\\Windows\\fonts\\'
+            - 'C:\\Windows\\help\\'
+            - 'C:\\Windows\\drivers\\'
+            - 'C:\\Windows\\addins\\'
+            - 'C:\\Windows\\cursors\\'
+            - 'C:\\Windows\\system32\tasks\\'
```

Comment:
these were `startswith` in the original rule, as well as that one on line 46. better keep it the same way or ask the author about the possibility to change it not breaking the original idea

Reply:
fixed